theforeman.org/pipelines/test/katello-pulpcore-integration.groovy

```diff
@@ -0,0 +1,22 @@
+pipeline {
+    agent none
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage('Test Integration') {
+            agent any
+
+            steps {
+                script {
+                    runCicoPipelines('katello_devel', 'devel', pipelines)
```

Comment:
given you have no `version.groovy`, you'll need to define `pipelines` somehere, like in this file:

```
def pipelines = [
    'katello_devel': [
        'centos7',
        'centos8'
    ]
]
```

but that should not be the reason why it fails